Modified useGit and workLocally to account for changes to Clone dialog (#131)
madewithkoji/docs: src/asciidoc/docs/develop/useGit.adoc

You should know your username by heart, and the access token is a 32-character, randomly-generated hexadecimal number, which you need to note in a safe place.

> If the project has a backend, you also need the values of the *`*KOJI_PROJECT_ID` and `KOJI_PROJECT_TOKEN` environment variables.

Reviewer: Add as the third bullet and simplify, e.g.
* If the project has a backend, the values of the `KOJI_PROJECT_ID`...

Clean up the errant asterisks before KOJI_PROJECT_ID.

Author: I'm not sure how you mean to simplify that line. I just removed "also".

Reviewer: I meant to use the same text I did, for parallel phrasing of the bullets. This change works as is, though.

WARNING: After you close the window, the access key will be displayed only by name (for example, `Token 1`).
You will not be able to retrieve it from the Koji site again.
You can, however, revoke your token and generate a new one if you have forgotten it, or if you believe that someone unauthorized has discovered it.
+

> NOTE: Keep the dialog open for the next two steps.

Reviewer: Remove this note. It seems more distracting than helpful, especially since you have clear instructions for how to reopen the dialog box where relevant.

=== Obtaining Koji environment variables

[NOTE]

> If your project does not have a backend, skip this section.

Reviewer: Again, we can remove this note. The intro already says that this section applies only if you have a backend.

> If your project has a backend, you need the values of the `KOJI_PROJECT_ID` and `KOJI_PROJECT_TOKEN` environment variables to access the leaderboard database.

Reviewer: ```suggestion
If your project has a backend, you need the values of the `KOJI_PROJECT_ID` and `KOJI_PROJECT_TOKEN` environment variables to access it.
```

Making this intro more generic, since not every project will have a "leaderboard" and a database is not the only backend service you might need to access.

> . If the Clone dialog is not already open, return to the browser tab for *Project Details*, and click *Clone*. The dialog opens.

Reviewer: ```suggestion
. If the Clone dialog is not already open, return to the browser tab for *Project Details*, and click *Clone*.
```
Removed because it's redundant.

. Click the first line of text under *Environment Variables* to highlight it.
. Copy the highlighted line to the clipboard.

> . Paste the line somewhere to save it for later.

Reviewer: These steps seem a bit simplistic, and don't really say what you're highlighting (which is the important part). Maybe try combining copy & paste into one step. Also, indicate what you're copying (e.g., first line is the project ID).

. Do the same with the second line of text.

=== Authorizing Git to clone the repository

> . If the Clone dialog is not already open, return to the browser tab for *Project Details*, and click *Clone*. The dialog opens.

Reviewer: ```suggestion
. If the Clone dialog is not already open, return to the browser tab for *Project Details*, and click *Clone*.
```
